programs/clearing_house/src/controller/insurance.rs

```diff
@@ -41,18 +41,18 @@ pub fn add_insurance_fund_stake(
         "Insurance Fund balance should be non-zero for new LPs to enter"
     )?;
 
-    let if_shares_before = insurance_fund_stake.if_shares;
-    let total_if_shares_before = spot_market.total_if_shares;
-    let user_if_shares_before = spot_market.user_if_shares;
-
     apply_rebase_to_insurance_fund(insurance_vault_amount, spot_market)?;
```

Comment:
I feel like semantically the rebase should be at the very top of all these functions, but it seems like it could potentially avoid future issues if it's always consistent like that.